docs/source/kale.loaddata.rst

```diff
@@ -126,6 +126,14 @@ kale.loaddata.videos module
    :exclude-members: VideoRecord
    :show-inheritance:
 
+kale.loaddata.n\_way\_k\_shot module
```

Comment:
N-way-k-shot is the official name describing this kind of problems, this module is designed to load any datasets with this problem formulation instead of focusing on a specific dataset. 

Given it dealing with images, is image_n_way_k_shot better?

Reply:
change to few_shot.py